lessons/Data-Science/ai-decision-trees/langs/en-us/index.adoc

```diff
@@ -0,0 +1,278 @@
+
+Computers, however, build decision trees using only the data we provide... and that data can sometimes be messy. As a result, we may end up with models that are not 100% accurate.
+
+Imagine that you are tasked with building a decision tree that can determine an iris' species (_setosa_, _virginica_, and _versicolor_) based on the varying plant measurements. It would be far more challenging to build this tree than the one we just made that classifies common tableware!
+
+@centered-image{images/iris-data.png}
+
+=== Synthesize
+
+@QandA{
+
+@Q{How is playing 20 questions to figure out a mystery item _similar_ to the predicting and recommending that a decision tree enables?}
+@A{Decision trees, like 20 Questions participants, need to avoid questions that don't provide information. Decision trees, like 20 Questions participants, sometimes fail to identify an object correctly. Familiarity with the dataset improves the accuracy of both a decision tree and a 20 Questions participant.}
+
+@Q{How is it different?}
+@A{When we play 20 Questions, we are trying to identify a single object. Decision trees are useful for categorizing a variety of objects, or predicting outcomes in a variety of different scenarios.}
+}
+
+
+
+== Decision Trees from Training Datasets @duration{25 minutes}
+
+=== Overview
+
+Students build a decision tree that predicts whether different individuals will purchase a video game or not.
+
+=== Launch
+
+We have already built some extremely simple decision trees. We have a sense of the hierarchical structure, flexibility, and versatility of decision trees. We know that decision trees sometimes fail when provided with ambiguous or unknown data. We understand the risk of overfitting our data (building a model that can only succeed on narrow set of data).
+
+We have *not* yet learned about how decision trees built from large datasets make relatively accurate predictions, recommendations, and diagnoses.
+
+The key to building an effective decision tree is to decide—at every level and every node—which attributes are the most informative ones to ask questions about.
+
+...but how do we make such decisions?!
+
+It turns out, there's an algorithm for that, and it's relatively straightforward.
+
+
+=== Investigate
+
+Have you ever done some online shopping—say, for a new pair of sneakers—only to discover that, for the next several days, you encounter _advertisements for sneakers_ lurking in every corner of the internet that you visit?!
+
+Is it a coincidence? No. Computer cookies are small data files stored locally on your device. One particular kind of cookie, the tracking cookie, allows AI designed for marketing to use your individual browsing habits to decide which ads you will be the most susceptible to.
```

Comment:
"Computer cookies" are not a thing. Suggest the following rephrasing:

"On a computer, 'cookies' are small data files that a website can store on your device. These can be used to remember where you were the last time you visited a site, or a setting that you changed and want to keep the next time you visit a site."

Reply:
Lol. The internet DEFINITELY thinks computer cookies are a thing.  But I am happy to accept your rephrasing.